tests/Oryx.Tests.Common/TestProgrammingPlatform.cs

```diff
@@ -75,7 +79,7 @@ public IEnumerable<string> GetDirectoriesToExcludeFromCopyToBuildOutputDir(Build
             return Array.Empty<string>();
         }
 
-        public bool IsCleanRepo(ISourceRepo repo)
+        public bool IsCleanRepo(BuildScriptGenerator.ISourceRepo repo)
```

Comment:
nit: should we move `ISourceRepo `and `LocalSourceRepo `into `common` proj so that they're pointing to the same one?